packages/glib/build.sh

```diff
@@ -67,11 +66,62 @@ termux_step_host_build() {
 termux_step_pre_configure() {
 	# glib checks for __BIONIC__ instead of __ANDROID__:
 	CFLAGS+=" -D__BIONIC__=1"
+	_PREFIX=$TERMUX_PKG_TMPDIR/prefix
+	local _WRAPPER_BIN="${TERMUX_PKG_BUILDDIR}/_wrapper/bin"
+	rm -rf $_PREFIX $_WRAPPER_BIN
+	mkdir -p $_PREFIX $_WRAPPER_BIN
 
-	TERMUX_PKG_VERSION=. termux_setup_gir
+	sed '/^export PKG_CONFIG_LIBDIR=/s|$|:'${_PREFIX}'/lib/pkgconfig|' \
+		"${TERMUX_STANDALONE_TOOLCHAIN}/bin/pkg-config" \
+		> "${_WRAPPER_BIN}/pkg-config"
+	chmod +x "${_WRAPPER_BIN}/pkg-config"
+	export PKG_CONFIG="${_WRAPPER_BIN}/pkg-config"
+	export PATH="${_WRAPPER_BIN}:${PATH}"
+
+	# Magic happens here.
+	# I borrowed nested building method from https://github.com/termux/termux-packages/blob/1244c75380beefc7f7da9744d55aa88df1640acb/x11-packages/qbittorrent/build.sh#L21-L28
+	# and modified termux_step_configure_meson in runtime to make it use another prefix
+	# Also I used advice from here https://github.com/termux/termux-packages/issues/20447#issuecomment-2156066062
+
+	(
+		eval `declare -f termux_step_configure_meson | sed 's|\$TERMUX_PREFIX|'$_PREFIX'|g'`
+		TERMUX_PKG_BUILDDIR=$TERMUX_PKG_TMPDIR/glib-build
+		mkdir -p $TERMUX_PKG_BUILDDIR
+		TERMUX_PKG_EXTRA_CONFIGURE_ARGS="${TERMUX_PKG_EXTRA_CONFIGURE_ARGS/"-Dintrospection=enabled"/"-Dintrospection=disabled"}"
+		TERMUX_PKG_VERSION=. termux_setup_gir
+		
+		cd "$TERMUX_PKG_BUILDDIR"
+		termux_step_configure
+		cd "$TERMUX_PKG_BUILDDIR"
+		termux_step_make
+		cd "$TERMUX_PKG_BUILDDIR"
+		termux_step_make_install
+	)
+
+	(
```

Comment:
Please provide a comment at the top of each of these subshells for clarity.
Something along the lines of:
```suggestion
	( # Phase 2, do whatever we needed to shave those Yaks for.
```